es-core/src/resources/Font.cpp

```diff
@@ -610,42 +610,69 @@ float Font::getLetterHeight()
 
 //the worst algorithm ever written
 //breaks up a normal string with newlines to make it fit xLen
+// supported UTF-8 text (wordwrap and cheap hyphenation)
 std::string Font::wrapText(std::string text, float xLen)
```

Comment:
I know you did not name this, but xLen really is just a cryptic name for what should be called 'width' or maxWidth.